llama_recipes/configs/datasets.py

```diff
@@ -46,4 +46,8 @@ class completion:
     data_path: str = None
     num_validation_samples: int = 100
     run_validation: bool = True
-    validation_data_path: str = None
\ No newline at end of file
+    validation_data_path: str = None
+    pack_sequences: bool = True
```

Comment:
imo it'd be nice to expose at least `pack sequences` as an additional argument to `train.py`.

Reply:
I tested this out just now and got an error about a nonexistant loss value. Not entirely sure what that's about but also I'm ok tabling it for now (unless some magic solution shows up in the AM).

Reply:
Hmm, can you share a snippet so I can try to reproduce?

Reply:
Fixed. 

Inputs need a `labels` field, but the field was being populated during the packing process. So, when we turned off packing, `labels` were never created. 

I moved the `labels` creation step to the initial tokenization step, which fixed this.